cmd/wallet-web/src/components/DIDManagement/DIDPreferencesComponent.vue

```diff
@@ -56,6 +57,28 @@
       </div>
     </div>
 
+    <div class="flex w-full grow flex-col items-start justify-start">
+      <label for="proof-format-selector" class="mb-1 text-base">Update Proof Format:</label>
+      <div
+        v-for="proofFormat in allProofFormats"
+        id="proof-format-selector"
```

Comment:
Looks like every `div` in rendered in this loop will have the same id (`proof-format-selector`), which would cause error on the page (`id`s should be unique)

Reply:
Hmm, I made that change (and a similar one in signature-type-selector) to fix an issue that my IDE was complaining about, with labels referencing nonexistent IDs. But that warning was already there before my changes, so I guess it's not really any issue?